app/frontend/src/components/Avatar.tsx

```diff
@@ -12,7 +13,12 @@ export interface AvatarProps extends MuiAvatarProps {
   user: User.AsObject;
 }
 
-export default function Avatar({ user }: AvatarProps) {
+export default function Avatar({ user, className }: AvatarProps) {
   const classes = useStyles();
-  return <MuiAvatar className={classes.root} src={user.avatarUrl} />;
+  return (
+    <MuiAvatar
+      className={classNames([className, classes.root])}
```

Comment:
I think this also works.
```suggestion
      className={classNames(className, classes.root)}
```